--- settings.py	2020-02-06 18:28:45.637307286 +0000
+++ settings.target.py	2020-02-06 18:32:23.141193745 +0000
@@ -329,15 +329,15 @@
 
 # (";"-separated) part of value of attribute with key SHIB_ENTITLEMENT needed in order to be considered a valid user:
 # if SHIB_AUTH_ENTITLEMENT is empty no constraint on SHIB_ENTITLEMENT attribute value is enforced (can even be missing)
-SHIB_AUTH_ENTITLEMENT = 'urn:mace:example.com:pki:user'
-#SHIB_AUTH_ENTITLEMENT = '' # is also in settings_local.py so override it there
+#SHIB_AUTH_ENTITLEMENT = 'urn:mace:example.com:pki:user'
+SHIB_AUTH_ENTITLEMENT = '' # is also in settings_local.py so override it there
 
-SHIB_USERNAME = ['HTTP_EPPN'] # in settings.py.dist # originally used for SHIB_USERNAME
-SHIB_USERNAME_DISPLAY_NAME = "eduPersonPrincipalName"
-SHIB_USERNAME_DISPLAY_ADDINFO = "urn:mace:dir:attribute-def:eduPersonPrincipalName or urn:oid:1.3.6.1.4.1.5923.1.1.1.6"
-#SHIB_USERNAME = ['HTTP_PERSISTENT_ID']
-#SHIB_USERNAME_DISPLAY_NAME = "PERSISTENT_ID"
-#SHIB_USERNAME_DISPLAY_ADDINFO = "urn:oid:1.3.6.1.4.1.5923.1.1.1.10"
+#SHIB_USERNAME = ['HTTP_EPPN'] # in settings.py.dist # originally used for SHIB_USERNAME
+#SHIB_USERNAME_DISPLAY_NAME = "eduPersonPrincipalName"
+#SHIB_USERNAME_DISPLAY_ADDINFO = "urn:mace:dir:attribute-def:eduPersonPrincipalName or urn:oid:1.3.6.1.4.1.5923.1.1.1.6"
+SHIB_USERNAME = ['HTTP_PERSISTENT_ID']
+SHIB_USERNAME_DISPLAY_NAME = "PERSISTENT_ID"
+SHIB_USERNAME_DISPLAY_ADDINFO = "urn:oid:1.3.6.1.4.1.5923.1.1.1.10"
 
 SHIB_SLUGIFY_USERNAME = False # value in settings.py.dist
 #SHIB_SLUGIFY_USERNAME = True
@@ -347,13 +347,13 @@
 SHIB_MAIL_DISPLAY_ADDINFO = "urn:mace:dir:attribute-def:mail or urn:oid:0.9.2342.19200300.100.1.3 or SHIB_INETORGPERSON_MAIL"
 
 # both optional:
-SHIB_FIRSTNAME = ['HTTP_SHIB_INETORGPERSON_GIVENNAME'] # value in settings.py.dist
-#SHIB_FIRSTNAME = ['HTTP_GIVENNAME']
+#SHIB_FIRSTNAME = ['HTTP_SHIB_INETORGPERSON_GIVENNAME'] # value in settings.py.dist
+SHIB_FIRSTNAME = ['HTTP_GIVENNAME']
 SHIB_FIRSTNAME_DISPLAY_NAME = "GIVENNAME"
 SHIB_FIRSTNAME_DISPLAY_ADDINFO = "urn:mace:dir:attribute-def:givenName or urn:oid:2.5.4.42"
 
-SHIB_LASTNAME = ['HTTP_SHIB_PERSON_SURNAME'] # value in settings.py.dist
-#SHIB_LASTNAME = ['HTTP_SURNAME']
+#SHIB_LASTNAME = ['HTTP_SHIB_PERSON_SURNAME'] # value in settings.py.dist
+SHIB_LASTNAME = ['HTTP_SURNAME']
 SHIB_LASTNAME_DISPLAY_NAME = "SURNAME"
 SHIB_LASTNAME_DISPLAY_ADDINFO = "urn:mace:dir:attribute-def:sn or urn:oid:2.5.4.4"
 
