--- settings.py.dist	2022-05-18 09:27:15.260147119 +0000
+++ settings.py	2022-05-18 12:10:24.085777698 +0000
@@ -33,7 +33,7 @@
 ##############################################################################
 ##############################################################################
 
-DEBUG = False
+DEBUG = True
 TEMPLATE_DEBUG = DEBUG
 
 ##
@@ -57,8 +57,8 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.',  # DB_ENGINE # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': '',                       # DB_NAME
+        'ENGINE': 'django.db.backends.sqlite3',  # DB_ENGINE # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
+        'NAME': 'example-data',                  # DB_NAME
         'USER': '',                       # DB_USER
         'PASSWORD': '',                   # DB_PASSWORD
         'HOST': '',                      # Set to empty string for localhost.
@@ -81,25 +81,20 @@
 LOGGING = {
     'version': 1,
     'disable_existing_loggers': False,
-    'filters': {
-        'require_debug_false': {
-            '()': 'django.utils.log.RequireDebugFalse'
-        }
-    },
     'handlers': {
-        'mail_admins': {
-            'level': 'ERROR',
-            'filters': ['require_debug_false'],
-            'class': 'django.utils.log.AdminEmailHandler'
-        }
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': 'debug.log',
+        },
     },
     'loggers': {
-        'django.request': {
-            'handlers': ['mail_admins'],
-            'level': 'ERROR',
+        'django': {
+            'handlers': ['file'],
+            'level': 'INFO',
             'propagate': True,
         },
-    }
+    },
 }
 
 ##############################################################################
@@ -372,7 +367,7 @@
 # email config
 
 # Notifications
-DISABLE_EMAIL_NOTIFICATION = True
+DISABLE_EMAIL_NOTIFICATION = False
 SERVER_EMAIL = "Example FoD Service <noreply@example.com>"
 EMAIL_SUBJECT_PREFIX = "[FoD] "
 #DEFAULT_FROM_EMAIL = "myemail@gmail.com"
@@ -629,7 +624,7 @@
 ##############################################################################
 # special test installation 
 
-ENABLE_SETUP_VIEW = False
+ENABLE_SETUP_VIEW = True
 
 ##############################################################################
 ##############################################################################
