--- settings.py.dist	2020-05-27 16:04:09.459076296 +0200
+++ settings.py	2020-05-27 16:06:59.378164956 +0200
@@ -30,7 +30,7 @@
 import os
 from celery.schedules import crontab
 
-DEBUG = False
+DEBUG = True
 TEMPLATE_DEBUG = DEBUG
 
 ADMINS = (
@@ -51,8 +51,8 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.',  # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': '',
+        'ENGINE': 'django.db.backends.sqlite3',  # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
+        'NAME': 'example-data',
         'USER': '',
         'PASSWORD': '',
         'HOST': '',                      # Set to empty string for localhost.
@@ -195,25 +195,20 @@
 LOGGING = {
     'version': 1,
     'disable_existing_loggers': False,
-    'filters': {
-        'require_debug_false': {
-            '()': 'django.utils.log.RequireDebugFalse'
-        }
-    },
     'handlers': {
-        'mail_admins': {
-            'level': 'ERROR',
-            'filters': ['require_debug_false'],
-            'class': 'django.utils.log.AdminEmailHandler'
-        }
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': 'debug.log',
+        },
     },
     'loggers': {
-        'django.request': {
-            'handlers': ['mail_admins'],
-            'level': 'ERROR',
+        'django': {
+            'handlers': ['file'],
+            'level': 'INFO',
             'propagate': True,
         },
-    }
+    },
 }
 
 LOGIN_URL = '/welcome'
@@ -292,7 +287,7 @@
 CELERY_IMPORTS = ("flowspec.tasks", )
 
 # Notifications
-DISABLE_EMAIL_NOTIFICATION = True
+DISABLE_EMAIL_NOTIFICATION = False
 SERVER_EMAIL = "Example FoD Service <noreply@example.com>"
 EMAIL_SUBJECT_PREFIX = "[FoD] "
 EXPIRATION_NOTIFY_DAYS = 4
@@ -491,7 +486,7 @@
     'LANGUAGE_CODE',
 ]
 
-ENABLE_SETUP_VIEW = False
+ENABLE_SETUP_VIEW = True
 
 from flowspy.settings_local import *
 
